pdc/apps/release/views.py

```diff
@@ -659,11 +659,16 @@ def create(self, request):
                 "component_dist_git_branch":    string,     # optional
                 "include_inactive":             bool,       # optional
             }
+
         __Response__:
+
             {
             "url": [the link for target release component]
             }
 
+        If the source-release doesn't contains components, which will return
+        400  {'detail': 'there is no component in source release'}.
```

Comment:
Is it better to change the description to
"Please make sure source release contains components, cloning source release without components doesn't make much sense."